NEWS.md

```diff
@@ -22,32 +22,87 @@ OpenSSL 3.2
-  * Multiple new features and improvements of the CMP protocol support.
-  * Subject or issuer names in X.509 objects are now displayed as UTF-8 strings
-    by default.
-  * TCP Fast Open (RFC7413) support is available on Linux, macOS, and FreeBSD
-    where enabled and supported.
+OpenSSL 3.2.0 is a feature release adding significant new functionality to
+OpenSSL.
+
+This release incorporates the following potentially significant or incompatible
+changes:
+
   * The default SSL/TLS security level has been changed from 1 to 2.
-  * Full support for provider-based/pluggable signature algorithms in TLS 1.3
-    operations as well as CMS and X.509 data structure support. With a suitable
-    provider this fully enables use of post-quantum/quantum-safe cryptography.
-  * It is now possible to use the IANA standard names in TLS cipher
-    configuration.
+
   * The `x509`, `ca`, and `req` apps now always produce X.509v3 certificates.
-  * Support for Argon2d, Argon2i, Argon2id KDFs has been added along with
-    a basic thread pool implementation for select platforms.
+
+  * Subject or issuer names in X.509 objects are now displayed as UTF-8 strings
+    by default.
+
+This release adds the following new features:
+
+  * Support for client side QUIC, including support for
+    multiple streams (RFC 9000)
+
+  * Support for Ed25519ctx, Ed25519ph and Ed448ph in addition
+    to existing support for Ed25519 and Ed448 (RFC 8032)
+
+  * Support for deterministic ECDSA signatures (RFC 6979)
+
+  * Support for AES-GCM-SIV, a nonce-misuse-resistant AEAD (RFC 8452)
+
+  * Support for the Argon2 KDF, along with supporting thread pool
+    functionality (RFC 9106)
+
+  * Support for Hybrid Public Key Encryption (HPKE) (RFC 9180)
+
+  * Support for SM4-XTS
+
+  * Support for Brainpool curves in TLS 1.3
+
+  * Support for TLS Raw Public Keys (RFC 7250)
+
+  * Support for TCP Fast Open on Linux, macOS and FreeBSD,
+    where enabled and supported (RFC 7413)
+
+  * Support for TLS certificate compression, including library
+    support for zlib, Brotli and zstd (RFC 8879)
+
+  * Support for provider-based pluggable signature algorithms
+    in TLS 1.3 with supporting CMS and X.509 functionality
+
+    With a suitable provider this enables the use of post-quantum/quantum-safe
+    cryptography.
+
+  * Support for using the Windows system certificate store as a source of
+    trusted root certificates
+
+    This is not yet enabled by default and must be activated using an
+    environment variable. This is likely to become enabled by default
+    in a future feature release.
+
+  * Support for using the IANA standard names in TLS ciphersuite configuration
+
+  * Multiple new features and improvements to CMP protocol support
+
+The following known issues are present in this release and will be rectified
+in a future release:
+
+  * Provider-based signature algorithms cannot be configured using the
+    SignatureAlgorithms configuration file parameter (#22761)
```

Comment:
Should this be a link to the issue?

Reply:
I think this is a bit overkill. We would end up having to add a link for every issue for every bug fixed in every release and it would get excessive. It would also make it less readable for people viewing the file in a terminal which I think is important.

Reply:
Middle ground: *if* an issue is referred to, like here, then we should have a link to it.  It's not terribly hard either:

```suggestion
    SignatureAlgorithms configuration file parameter ([\#22761])
    
[\#22761]: https://github.com/openssl/openssl/issues/22761
```

Reply:
@levitte are you sure the backslash before the hash is really required inside the square brackets? Without it, it seems to work as well:
```
SignatureAlgorithms configuration file parameter ([#22761])
    
[#22761]: https://github.com/openssl/openssl/issues/22761
```

SignatureAlgorithms configuration file parameter ([#22761])
    
[#22761]: https://github.com/openssl/openssl/issues/22761

Reply:
No, not entirely sure...  mostly, I wanted to avoid unnecessary github interpretations

Reply:
Personally I am still against this. We can have these linked automatically once we autogenerate release notes on the website from this.